src/configurations/destinations/zoho_dev/accounts/zoho_dev_oauth/schema.json

```diff
@@ -0,0 +1,14 @@
+{
+  "configSchema": {
```

Comment:
Should we call this config schema or options schema in the definitions CRUD API we call this optionsSchema ref: https://docs.scalar.com/editor#/reference/model/createaccountdefinitionrequest